src/components/Button/Button.module.scss

```diff
@@ -4,6 +4,7 @@
   line-height: 1;
 
   &:not(.naked) {
+    display: inline-block;
```

Comment:
applying this because when rendering an anchor tag, we want it to have inline-block behavior.